packages/node_modules/@webex/plugin-meetings/test/integration/spec/journey.js

```diff
@@ -286,6 +297,80 @@ skipInNode(describe)('plugin-meetings', () => {
           throw e;
         }));
 
+      // FHD - remote video tests - starts
```

Comment:
not sure what "starts" is supposed to mean here

Reply:
This was for reference as file size grows and it becomes difficult to keep track. Removed.